hail/python/hail/experimental/ndarray.py

```diff
@@ -0,0 +1,16 @@
+import hail as hl
+import numpy as np
+import hail.ir as ir
+from hail.expr import construct_expr, to_expr
+
+from hail.typecheck import typecheck
+
+
+@typecheck(nd=np.ndarray)
+def ndarray(nd):
+    nd = nd.astype(np.float64)
+    data = hl.array(list(nd.flat))
```

Comment:
Did you want this in the documentation? I presume not.

Reply:
Maybe this would be better as an '_' method in `functions.py`.

Reply:
Ya definitely hidden for now, especially since NDArrays are not interpretable and don't work unless you have a spark backend with c++ enabled. Moved it as an `_` function in `functions.py`